packages/p2p/src/stores/general-store.js

```diff
@@ -204,6 +204,9 @@ export default class GeneralStore extends BaseStore {
     showCompletedOrderNotification(advertiser_name, order_id) {
         const notification_key = `order-${order_id}`;
 
+        // we need to refresh notifications in notifications-store in the case of a bug where when user closes the notification, the notification count is not synced up with the closed notification
+        this.props.refreshNotifications();
```

Comment:
We need to refresh and re-filter notifications using the actions `refreshNotifications` and `filterNotificationMessages` within `notifications-store`, since in the case where the user closes the notification, we called `removeNotificationMessage` which filters out the closed notification from the observable `notification_messages`, and thus when another completed order is shown and the `filtered_notifications` is re-computed, the `notifications` observable is filtered out according to `notification_messages` which excludes the closed notification that the user closed just now. Steps to perform:
1. User completes an order and the completed order notification is shown, the notification count is reflecting the new completed order notification
2. User closes the completed order notification by clicking X button
3. When user completes another order, the completed notification order is shown, but the notification count does not include and reflect the new completed order